### PR TITLE
Fix minor grammar and typos - part 2

### PR DESCRIPTION
**Pull request by tcharding**
This PR follows on from the work done in [part 1](https://github.com/filecoin-project/specs/pull/917).

Minor grammar fixes and typos/spelling fixes for the second half of the spec (2.4.4 until finish).

As part of this work I raised a few issues about the HTML rendering.

A few things I did _not_ fix/change

- Full stop usage in lists
- Capitalization of terms that exist in Go code i.e., `Cid`
- Capitalization of 'golang'
- Uniform usage of abbreviations: i.e. (ie) and e.g. (eg) 

This PR is made up of many single line fixes to facilitate dropping any of them if required.

Good work on the spec, it is very well written.

Thanks

### CODE REVIEW

**Comment by tcharding** [comment]
If this passes review I can squash into a single commit and rebase.  Thanks.

File: src/glossary/_index.md [hunk 1]
```diff
@@ -265,10 +265,10 @@ Something that is verifiable can be checked for correctness by a third party.
 
 ## VDF
 
-A verifiable function that guarantees a time delay given some hardware assumptions and a small set of requirements. These requirements are efficient proof verification, random output, and strong sequentiality. Verifiable delay functions are formally defined by [\[BBBF]](https://eprint.iacr.org/2018/601).
+A verifiable function that guarantees a time delay given some hardware assumptions and a small set of requirements. These requirements are efficient proof verification, random output, and strong sequentiality. Verifiable delay functions are formally defined by [BBBF](https://eprint.iacr.org/2018/601).
```
Comment (sternhenri) [inline src/glossary/_index.md:268]: This is an explicit paper ref
```suggestion
A verifiable function that guarantees a time delay given some hardware assumptions and a small set of requirements. These requirements are efficient proof verification, random output, and strong sequentiality. Verifiable delay functions are formally defined by [\[BBBF]](https://eprint.iacr.org/2018/601).
```

File: src/algorithms/gossip_sub.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "GossipSub"
 
 {{<label gossip_sub>}}
 
-Messages and block headers along side the message references are propagated using the [gossipsub libp2p pubsub router](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub). Every full node must implement and run that protocol. All pubsub messages are authenticated and must be [syntactically validated](./validation.md#syntactical-validation) before being propagated further.
+Messages, and block headers along side the message references, are propagated using the [gossipsub libp2p pubsub router](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub). Every full node must implement and run that protocol. All pubsub messages are authenticated and must be [syntactically validated](./validation.md#syntactical-validation) before being propagated further.
```
Comment (sternhenri) [inline src/algorithms/gossip_sub.md:7]: harder to read with commas imo 

```suggestion
Messages and block headers along side the message references are propagated using the [gossipsub libp2p pubsub router](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub). Every full node must implement and run that protocol. All pubsub messages are authenticated and must be [syntactically validated](./validation.md#syntactical-validation) before being propagated further.
```

File: src/systems/filecoin_markets/storage_market/_index.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ Negotiation begins once a client has discovered a miner whose `StorageAsk` meets
 
 Execution now moves to the `StorageProvider`
 
-4. The `StorageProvider` inspects the deal to verify that its parameters match it own internal criteria (such as price, piece size, deal duration, etc) -- and rejects the proposal if it doesn't (it sends a rejection to the client over the
+4. The `StorageProvider` inspects the deal to verify that the deals parameters match its own internal criteria (such as price, piece size, deal duration, etc) -- and rejects the proposal if it doesn't (it sends a rejection to the client over the
```
Comment (sternhenri) [inline src/systems/filecoin_markets/storage_market/_index.md:64]:
```suggestion
4. The `StorageProvider` inspects the deal to verify that the deal's parameters match its own internal criteria (such as price, piece size, deal duration, etc) -- and rejects the proposal if it doesn't (it sends a rejection to the client over the
```

**Pull request merged**